tests/asynctests/combined_asynctests.js

```diff
@@ -0,0 +1,44 @@
+
+function testAll() 
+{
+
+    api.getIntValueAsync( function (val) {
```

Comment:
Please comply to the Qt coding conventions (with regard to spaces: "api.getIntValueAsync(function(val) {"). That's what we agreed on in this project. Here and all over the patch.